docker/images/visor/Dockerfile

```diff
@@ -35,3 +39,9 @@ ENTRYPOINT [ "./skywire-visor" ]
 
 # default target
 FROM  visor-runner
+
+LABEL org.opencontainers.image.created="${BUILD_DATE}" \
```

Comment:
What do we need those labels for?

Reply:
Just to help organize things perhaps? It's not very useful right now I know. I'll remove it